build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -4,12 +4,26 @@
   <PropertyGroup>
     <_SourceTopDir>..\..</_SourceTopDir>
     <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MandroidInternalDir>$(OutputPath)\lib\mandroid-internal</_MandroidInternalDir>
```

Comment:
What's our end goal here, and should this PR wait until we're closer to that end goal?

I see two ways to proceed:

1. Add a git submodule for https://github.com/mono/api-doc-tools, and build `mdoc.exe` from source so that it references our `Xamarin.Android.Cecil.dll` assembly. (This would likely require additional changes to the `api-doc-tools` repo.)

2. Build `mdoc.exe` as part of the Mono build, then alter the `Mono.Cecil` assembly reference to `Xamarin.Android.Cecil`, via `remap-assembly` (which would also need to be imported).

I'm partial to (1), though (2) is certainly the *easier* answer, but if we want to do (2), shouldn't the introduction of `remap-assembly` and associated fixup be part of this PR?